packages/composite-checkout-wpcom/tsconfig.json

```diff
@@ -2,7 +2,8 @@
 	"compilerOptions": {
 		"allowJs": true,
 		"jsx": "react",
-		"target": "es5",
+		"target": "ES5",
+		"lib": [ "DOM", "ES2015", "ScriptHost" ],
```

Comment:
@nbloomf This seem OK to you?

My understanding is that target ES5 changes compiled output, while lib allows all ES2015 features, e.g. `Promise`, `Set`.